server_admin/topics/users/con-user-impersonation.adoc

```diff
@@ -14,7 +14,7 @@ image:{project_images}/user-details.png[]
 * If the administrator and the user are in the same realm, then the administrator will be logged out and automatically logged in as the user being impersonated.  
 * If the administrator and user are not in the same realm, the administrator will remain logged in, and additionally will be logged in as the user in that user's realm.  
 
-In both instances, the browser redirects to the impersonated user's *User Account Management* page. 
+In both instances, the *User Account Management* page of the impersonated user opens.
```

Comment:
We agree on "is displayed" where you have "opens."  Does that make sense.  I think the IBM style for "opens" is used when new window appears.